### DISCUSSION TITLE
[@types/howler] Property 'autoUnlock' does not exist on type 'typeof HowlerGlobal'.

### DISCUSSION THREAD

**Discussion by AndreMicheletti** [DefinitelyTyped/DefinitelyTyped#59072] [Issues with a @types package] [answered]
Hello,

I have `howler` and `@types/howler` installed in my node_modules, and I'm getting this error on VS Code type checking on every Howler global property:

![image](https://user-images.githubusercontent.com/16908595/156386018-bda86c9c-102d-44d9-89ff-73db820146f9.png)

![image](https://user-images.githubusercontent.com/16908595/156386599-c04d055c-2a7d-4a1f-8f5c-76e31a260c87.png)

The code is simply importing `Howler` global and trying to set one of its properties.

```javascript
import { Howler } from 'howler';

@provide(AudioEngines.Howler)
@ccclass
export default class HowlerAudioEngine implements AudioEngineInterface {
  public initialize(settings: AudioSettings, tweenNode: cc.Node, debug = false): AudioEngineInterface {
    // ...
    Howler.autoUnlock = true;
   }
 }
```

This is my `tsconfig.json`
```json
{
  "compilerOptions": {
    "module": "commonjs",
    "lib": ["dom", "dom.iterable", "esnext"],
    "target": "es6",
    "experimentalDecorators": true,
    "skipLibCheck": true,
    "forceConsistentCasingInFileNames": true
  },
  "exclude": ["node_modules", "library", "local", "temp", "build", "settings"]
}
```

Could you help me with that, please?

[a comment by an automated account is not shown]

**Comment by MrGriefs** [comment, marked as answer]
This is an issue generated from PR #58305 due to an overlook on my part.  
I'm waiting for my PR which fixes this to be reviewed, until then, I recommend using the prior declaration `npm i @types/howler@2.2.4`

**Reply by AndreMicheletti** [reply]
Great, thks!
